setup.py

```diff
@@ -1,35 +1,40 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 
 
 def readme():
-  with open('README.md') as f:
-    return f.read()
+    with open('README.md') as f:
+        return f.read()
 
 
 # Get version
-exec(open('pybrake/version.py').read())
+exec(open('src/pybrake/version.py').read())
 
-
-setup(name='pybrake',
-      version=version,
-      description='Python exception notifier for Airbrake',
-      long_description=readme(),
-      long_description_content_type='text/markdown',
-      classifiers=[
+setup(
+    name='pybrake',
+    version=version,
+    description='Python exception notifier for Airbrake',
+    long_description=readme(),
+    long_description_content_type='text/markdown',
+    classifiers=[
         'Development Status :: 3 - Alpha',
         'License :: OSI Approved :: MIT License',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
         'Topic :: Software Development',
-      ],
-      keywords='airbrake exception error notifier',
-      url='http://github.com/airbrake/pybrake',
-      author='Vladimir Mihailenco',
-      author_email='vlad@airbrake.io',
-      license='MIT',
-      packages=['pybrake'],
-      install_requires=['tdigest'],
-      tests_require=['tdigest'],
-      include_package_data=True,
-      zip_safe=False)
+    ],
+    keywords='airbrake exception error notifier',
+    url='http://github.com/airbrake/pybrake',
+    author='Vladimir Mihailenco',
+    author_email='vlad@airbrake.io',
```

Comment:
Would be nice to update this.

Reply:
Whose information should be added?

Reply:
https://github.com/airbrake/airbrake-ruby/blob/e01f2cd5a211cf00dc44d24826233884ad293a52/airbrake-ruby.gemspec#L17-L19